lib/puppet-syntax/tasks/puppet-syntax.rb

```diff
@@ -5,9 +5,11 @@
 module PuppetSyntax
   class RakeTask < ::Rake::TaskLib
     def filelist(paths)
+      excludes = PuppetSyntax.exclude_paths
+      excludes.push('pkg/**/*')
```

Comment:
Do we want to parse .gitignore and add all entries to the excludes list?

Reply:
Unless we think that other dirs will contain puppet manifest files, then it's probably more work for little benefit. That said, maybe we need to add an entry for `spec/**/*` as well.